lib/view/main_view/home/omni_website.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_webview_plugin/flutter_webview_plugin.dart';
+
+class WebViewPage extends StatefulWidget {
+  @override
+  _WebViewPageState createState() => _WebViewPageState();
+}
+class _WebViewPageState extends State<WebViewPage> {
+  @override
+  Widget build(BuildContext context) {
+    return WebviewScaffold(
+      url: "https://www.omnigroup.com/",
```

Comment:
This website is not affiliate/related to omni. it is a completely different group. 
Omni is http://www.omnilayer.org/

Reply:
Fixed